packages/nuxt/src/app/composables/state.ts

```diff
@@ -42,6 +42,8 @@ export function clearNuxtState (
 ): void {
   const nuxtApp = useNuxtApp()
   const _allKeys = Object.keys(nuxtApp.payload.state)
+    .map(key => key.substring(useStateKeyPrefix.length))
```

Comment:
I would also welcome a future PR that improves perf by not enumerating state keys when an array of strings is provided as the keys arg.